### PR TITLE
Change object_size method

### PR DESCRIPTION
**Pull request by SaravShah**
- Used updated moab-versioning gem for object_size method

connects to #61

### CODE REVIEW

**Comment by ndushay** [comment]
I just updated the tolerances on coveralls configuration to be 0.5%

File: Gemfile.lock [hunk 1]
```diff
@@ -55,7 +55,7 @@ GEM
       bundler (~> 1.2)
       thor (~> 0.18)
     byebug (9.1.0)
-    capistrano (3.9.0)
+    capistrano (3.9.1)
```
Comment (ndushay) [inline Gemfile.lock]: It's a good idea to split out gem updates into a commit separate from coding updates

Also, I would prefer if you update gems in separate commits of logical groupings, e.g. "update rails to 5.1.4" "update capistrano to 3.9.1" "update rubocop-rspec to 0.16.1" etc.

This is for two reasons:  

1. to easily see what has been updated in the commit history (when you are hunting for "what changed?  why isn't it working now?")  and  
2.  to make it easier to back out of some updates and refactor the PR if there are objections to some of the gem updates

File: app/controllers/moab_storage_controller.rb [hunk 2]
```diff
@@ -13,8 +13,7 @@ def index
   end
 
   def show
-    moab_path = path_from_object_id(params['id'])
-    object_size = directory_size(moab_path)
+    object_size = Stanford::StorageServices.object_size(params['id'])
```
Comment (ndushay) [inline app/controllers/moab_storage_controller.rb]: yay!  smaller code here!

File: spec/models/preserved_object_spec.rb [hunk 3]
```diff
@@ -11,7 +11,7 @@
   end
 
   it 'is not valid without a specified current_version' do
-    expect(PreservedObject.new(druid: 'ab123cd45678', preservation_policy: 'keepit', size: 1))
+    expect(PreservedObject.new(druid: 'ab123cd45678', preservation_policy: 'keepit', size: 1)).not_to be_valid
```
Comment (ndushay) [inline spec/models/preserved_object_spec.rb]: I just noticed this myself while working on my ticket.  This is another thing that should be a separate commit.  As I am working on a PR right now refactoring this spec a bit, I suggest you leave this off altogether and see if you're comfy with how I do it in my PR.

Comment (ndushay) [reply spec/models/preserved_object_spec.rb]: if my home connection will ever upload my changes ...  (See #74)

File: Gemfile.lock (hunk 1 above)
Comment (SaravShah) [reply Gemfile.lock]: So can I split this commit into 2 commits? 
1. Update Gem versions - i'm updating ~30 gems, and it seems like that would be a lot of commits if I were to create a commit for each gem. I may not be understanding you though. 

I could also just update Moab Versioning Gem and have that as 1 commit and not update anything else?

2. Update object size method

File: spec/models/preserved_object_spec.rb (hunk 3 above)
Comment (SaravShah) [reply spec/models/preserved_object_spec.rb]: Sounds good.

File: Gemfile.lock (hunk 1 above)
Comment (ndushay) [reply Gemfile.lock]: you can `bundle update rails`  which will take care of most of the gems as one commit.   Another commit after `bundle update moab-versioning` etc.

Or you can leave the non-moab-versioning updates off;  I can prepare a PR for those, if you like.

Comment (SaravShah) [reply Gemfile.lock]: sure. that works for me. and then this PR will just have 1 commit having to do with updating the object size. Sounds good.

**Comment by jmartin-sul** [comment]
it appears that @ndushay's concern has been addressed (by breaking out gem updates into #79), so i'm going to go ahead and merge this.
